plugins/org.jkiss.dbeaver.ext.mysql.ui/src/org/jkiss/dbeaver/ext/mysql/ui/views/MySQLConnectionPage.java

```diff
@@ -220,11 +223,18 @@ private void updateUrl() {
     public boolean isComplete() {
         if (isCustomURL()) {
             return !CommonUtils.isEmpty(urlText.getText());
+        } else {
+            boolean hostAndPortIsPresent = hostText != null &&
+                !CommonUtils.isEmpty(hostText.getText()) &&
+                (portText == null || !CommonUtils.isEmpty(portText.getText()));
+            DBPAuthModelDescriptor selectedAuthModel = authModelSelector.getSelectedAuthModel();
```

Comment:
Let's move this verification (isHostRequiredByAuthModel) to some utils class to avoid copy-paste.